### DISCUSSION TITLE
how to change the font size of folder tab

### DISCUSSION THREAD

**Discussion by pengyanj** [doublecmd/doublecmd#1925] [General]
Notice that the font is smaller than default size
I am wondering how to increase the font size of tabs?
Thanks!
<img width="112" alt="2024-10-16_08-01-20" src="https://github.com/user-attachments/assets/5853eb36-9e95-4f93-b38e-6d86d659fa07">

**Comment by alexx2000** [comment]
Only by changing system font size (in Windows settings).
